### PR TITLE
Separated Docker builds into submodules.

### PR DESCRIPTION
**Pull request by gregjan**
Added a Cassandra Docker image for performing Trellis schema init.
Docker builds copy all file dependencies from parent or webapp builds.
Docker builds only activated with the 'docker' profile is activated.
Removed all Docker-related things from webapp submodule.

adds JMS EventService implementation

commented out JMS eventservice in pom

### CODE REVIEW

File: webapp/pom.xml [hunk 1]
```diff
@@ -112,8 +112,15 @@
       </exclusions>
     </dependency>
 
+    <!-- Adds JMS-based EventService implementation
```
Comment (ajs6f) [inline webapp/pom.xml]: Hm. Not sure what this is doing in a Docker-related PR?

**Comment by ajs6f** [comment]
Hm, Travis failed, I'm going to just rerun it. Looks like some kind of weird class-not-found thing.

**Comment by gregjan** [comment]
I will create them READMEs, thank you.

**Comment by gregjan** [comment]
Huh, missing some kind of Jena RIOTException class, during testing.. I wonder where it went.

**Comment by ajs6f** [comment]
Yeah that's weird, because that is a very basic low-level type in Jena, and it's failing in Java 8, so not  JPMS oddity.

**Comment by acoburn** [comment]
Can you try using Jena 3.11.0? I recently updated that in the main code
base.

**Comment by ajs6f** [comment]
@acoburn is definitely correct. I remember now that I had to make the same change in the "new data layout for Mementos" branch.

**Comment by gregjan** [comment]
I'll do that straight away.

**Comment by ajs6f** [comment]
@gregjan Given how orthogonal our two PRs are, you might try just rebasing onto mine. Then I'll merge mine and then yours, and then there's only one commit with the change to Jena being made.

**Comment by gregjan** [comment]
Okay, I did a rebase and squashed my commits. The default tests went fine locally.

**Pull request merged**